packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -27,18 +27,18 @@ export class RetriableDocumentStorageService extends DocumentStorageServiceProxy
         return this.readWithRetry(async () => this.internalStorageService.getSnapshotTree(version));
     }
 
+    /**
+     *
+     * @deprecated - only here for back compat, will be removed after release
+     */
     public async read(blobId: string): Promise<string> {
-        return this.readWithRetry(async () => this.internalStorageService.read(blobId));
```

Comment:
I'd rather see code like that calling underlying storage directly, i.e. 
      return this.readWithRetry(async () => this.internalStorageService.rea(id));

Reason for this - if underlying storage works with base64 and decodes it to buffer when readBlob() call is made, then we add conversion back to base64, thus making 2 conversions for nothing (from CPU workload perspective). But if we call directly existing read() API, then potentially we are avoiding these costs.

Reply:
That said, it means that we keep read() API on storage interface for now. I think it's better that way, as we definitely do not want to regress perf anywhere, and we will (if we do extra conversions), at least until storage starts to send us binary data (right now it sends us base64 encoded data).

Reply:
ok! I also added some other implementations of read back since it's on the interface.